tests/pages/enterprise-banner.js

```diff
@@ -0,0 +1,26 @@
+import PageObject from 'travis/tests/page-object';
+
+let {
+  isHidden,
+  text
+} = PageObject;
+
+export const enterpriseBanners = {
+  trialBanner: PageObject.create({
+    scope: '.enterprise-banner-trial',
+    text: text(),
+    isHidden: isHidden()
+  }),
+
+  licenseBanner: PageObject.create({
+    scope: '.enterprise-license-banner',
+    text: text(),
+    isHidden: isHidden()
+  }),
+
+  seatsBanner: PageObject.create({
+    scope: '.enterprise-banner-seats',
+    text: text(),
+    isHidden: isHidden()
+  })
```

Comment:
`ember-cli-page-object` has a lightweight [component](http://ember-cli-page-object.js.org/docs/v1.13.x/components) concept that you could use here instead, it includes `isHidden` and `text` by default. So this last one could become this:

```
seatsBanner: {
  scope: '.enterprise-banner-seats'
}
```

Reply:
Good to know – will update this.